examples/src/main/java/com/example/ftclibexamples/MecanumDrivingSample.java

```diff
@@ -1,61 +1,57 @@
 package com.example.ftclibexamples;
 
 import com.arcrobotics.ftclib.drivebase.MecanumDrive;
-import com.arcrobotics.ftclib.geometry.Vector2d;
-import com.arcrobotics.ftclib.hardware.motors.MotorEx;
+import com.arcrobotics.ftclib.gamepad.GamepadEx;
+import com.arcrobotics.ftclib.hardware.RevIMU;
+import com.arcrobotics.ftclib.hardware.motors.Motor;
 import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
 
+@TeleOp
 public class MecanumDrivingSample extends LinearOpMode {
 
-    private MecanumDrive driveTrain;
-
-    private MotorEx fL, fR, bL, bR;
-
-    // the motors used here are goBILDA Yellow Jackets, 435 rpm
+    static final boolean FIELD_CENTRIC = false;
 
     @Override
     public void runOpMode() throws InterruptedException {
-
-        fL = new MotorEx(hardwareMap, "frontLeft", MotorEx.GoBILDA.RPM_435);
-        fR = new MotorEx(hardwareMap, "frontRight", MotorEx.GoBILDA.RPM_435);
-        bL = new MotorEx(hardwareMap, "backLeft", MotorEx.GoBILDA.RPM_435);
-        bR = new MotorEx(hardwareMap, "backRight", MotorEx.GoBILDA.RPM_435);
-
-        driveTrain = new MecanumDrive(
-                fL, fR, bL, bR
+        // constructor takes in frontLeft, frontRight, backLeft, backRight motors
+        // IN THAT ORDER
+        MecanumDrive drive = new MecanumDrive(
+            new Motor(hardwareMap, "frontLeft", Motor.GoBILDA.RPM_435),
+            new Motor(hardwareMap, "frontRight", Motor.GoBILDA.RPM_435),
+            new Motor(hardwareMap, "backLeft", Motor.GoBILDA.RPM_435),
+            new Motor(hardwareMap, "backRight", Motor.GoBILDA.RPM_435)
         );
 
-        waitForStart();
-
-        driveWithVector(new Vector2d(12, 3));
-        sleep(1000);
-        driveWithVector(new Vector2d(0, 0));
+        RevIMU imu = new RevIMU(hardwareMap);
+        imu.init();
```

Comment:
explain why we need the IMU and how it has to be oriented specifically maybe

Reply:
Should we fully explain the orientation of the IMU and why that matters? And how would we best go about explaining that without digressing from the actual point of the sample (to show how to use the mecanum drive)?

Reply:
Thats true, I just thought it might be best to fully explain each part of the sample so that its a sample and not just a showcase if you get what I mean.  Rather than just explaining one thing then needing to go to another example for one part of this one and then keep switching.